shell/packages/sandstorm-db/db.js

```diff
@@ -492,6 +500,15 @@ ApiTokens = new Mongo.Collection("apiTokens", collectionOptions);
 //         address :Text;
 //       }
 //       identity :Text;
+//       http :group {
+//         url :Text;
+//         auth :union {
+//           none :Void;
+//           bearer :Text;
+//           basic :group { username :Text; password :Text; }
```

Comment:
I wish we could encrypt these fields somehow. I don't see a good way to do that, though.